### DISCUSSION TITLE
Release date

### DISCUSSION THREAD

**Discussion by cesar404** [phase1geo/Minder#324] [General]
Hi. When will Minder 2.0 be released? Is there any release plan?

**Comment by phase1geo** [comment]
At the current rate, it might be a while before version 2.0 is ready.  I haven't been able to spend a lot of development time lately and my 2.0 feature list that needs to be done is somewhat lengthy.  I will create a GitHub project that will show the status of 2.0 development, however. That way, you should at least be able to see what is going on (and possibly influence features in that release as well).
